### PR TITLE
Custom domain: add a WebAuthn note

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds a note about re-enrolling in the WebAuthn authenticator when a new custom domain is configured.
- **Is this PR related to a Monolith release?** Nope.

### Resolves:

* [OKTA-521214](https://oktainc.atlassian.net/browse/OKTA-521214)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/8e9dde1a3d84e0133e226e12e648b6cfab3b5df8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | 80 | 123 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c28db545-0101-4daf-acda-3fe420091c02) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.56066a7c-6858-4b46-a987-c86beeaa0a09?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/2c7c4b00ff43b796c79e4c6440b34d814fda2a1e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | 79 | 125 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/890dc8f6-c7e8-44e9-80be-99885c6cce00) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.58988237-c1ce-413a-b999-e633256e7d86?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/caea9d74e28ca709617a2c4829f77bf1aab6c247/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | 80 | 123 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7c0df764-2995-40fd-b8c9-1b1ea0359afc) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.edc4e128-1735-461e-be9e-96c49f4cd851?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/28db4c4276bf55de2f41a159022a898e683973a0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | 80 | 123 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/af76295f-e45b-4295-ad8c-0499c636cb58) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.eed5257b-ef75-41f6-af1d-67b30fb1d8de?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by timhowie-okta — APPROVED** [review]
lgtm

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 1]
```diff
@@ -73,11 +73,13 @@ The third generation of the Okta Sign-In Widget doesn’t guarantee the stabilit
...
 
-* When you implement a custom domain, users aren't automatically rerouted from the original URL to the new custom URL. You must communicate the new custom domain to your users. One way to communicate the change is to [create a custom notification](https://help.okta.com/okta_help.htm?id=ext_Dashboard_End_User_Notifications) that appears on each user's dashboard.
+* When you implement a custom domain, users aren't automatically rerouted from the original URL to the new custom URL. You must communicate the new custom domain to your users.
+
+* If you configure the [FIDO2 (WebAuthn) authenticator](https://help.okta.com/okta_help.htm?type=oie&id=csh-configure-webauthn) in your org and create a custom domain, your users first need to re-enroll with another authenticator. For example, Okta Verify or email. Then, your users can re-enroll with WebAuthn. Communicate the new URL to your users so that Okta prompts them to re-enroll. Every domain a user accesses requires re-enrollment because each set of their credentials is scoped to a separate domain.
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:82]: Thank you. Will update.

**Pull request merged**